deploy/operator.yaml

```diff
@@ -443,6 +443,8 @@ rules:
   - endpoints
   - namespaces
   - services
+  - secrets
+  - configmaps
```

Comment:
This looks wider than necessary - see my other comment below.

Suggestion: on the helm template level, grant access to secrets on the per-resource level (ideally with `resourceNames`).

Reply:
duplicate of https://github.com/scylladb/scylla-operator/pull/2524#discussion_r2014115891